Makefile

```diff
@@ -142,11 +142,12 @@ dist: pipenv fetch-tools
 	-[ -n "$$NOSWITCH" ] || git checkout -
 
 fetch-tools:
-	-rm -Rf testsuite-tools/
-	-curl $(fetch_tools) | tar -xz
+	-rm -Rf ext/testsuite-tools/
+	-mkdir -p ext/
+	-curl $(fetch_tools) | tar -C ext/ -xz
```

Comment:
I don't think `curl |tar` return code should be ignored

Reply:
and I think it should :-)
It is ignored because it is ran everytime when building container, but tools url doesn't have to be available every time. Yes, whole tools target works just by "accident" and it isn't guaranteed. That's because it isn't guaranteed and purely optional. The rule is simple `make fetch-tools` (which should not be called explicitly) should never fail - should never block build process, rather build container with `make tools` not working